### PR TITLE
Fix: Single use of quiz room

### PR DESCRIPTION
**Pull request by HaroldCuico**
Added a condition to check if the room is already used, if it is used it should deny entering the room

* Display rooms that are not used
* Deny access to used rooms

### CODE REVIEW

**Review by jaymar921 — COMMENTED** [review]
nice new route, kindly update the session documentation

File: WebApp/backend/QuizMaster.API.QuizSession/Services/Grpc/QuizRoomServices.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.QuizSession/Services/Grpc/QuizRoomServices.cs:1]: nice implementation

File: WebApp/backend/QuizMaster.API.Gatewway/Services/QuizHandler.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Services/QuizHandler.cs:1]: nice new route to invoke, kindly update the session documentation

**Review by jaymar921 — APPROVED** [review]
approved

**Comment by jaymar921** [comment]
@kinloveko pls approve

**Review by kinloveko — APPROVED** [review]
Approved!

**Comment by kinloveko** [comment]
@jaymar921 @ha-rold1999  @HaroldCuico Approved!

**Pull request merged**